VirtualH89/Src/NilMemory8K.h

```diff
@@ -0,0 +1,29 @@
+///
+/// \file NilMemory8K.h
+///
+/// Implementation of an 8K page of non-existent memory. Reads 0, writes ignored.
+///
+/// \date Mar 05, 2016
+/// \author Douglas Miller
+///
+#ifndef NILMEMORY8K_H_
+#define NILMEMORY8K_H_
+
+#include "Memory8K.h"
+
+class NilMemory8K: public Memory8K
```

Comment:
One of my goals was to get as much of the readByte inlined as possible, since it is probably the single most called function in the entire emulator. This way, there is only one virtual function call in the entire read path - the one at AddressBus. I figured 8K was a small price.  Of course, this goal was based on past experience with C++ on HPC, so it may not turn out to be as important as I thought.